packages/trip-form/src/TripOptions/Checkbox.tsx

```diff
@@ -15,6 +15,10 @@ export interface ButtonProps {
 
 export default function Checkbox(
   props: {
+    checkboxIcons?: {
+      checked: FunctionComponent;
+      unchecked: FunctionComponent;
+    };
```

Comment:
This type is repeated at least 3 times and should be extracted.

Reply:
Good call thank you! Done in a10727e